<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encode/Decode Varint</string>
  </property>
  <widget class="QLineEdit" name="varIntLE">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>271</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Varint To Decode:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Integer to encode:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="intLE">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>271</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="decodeBTN">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Decode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encodeBTN">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>encode</string>
   </property>
  </widget>
  <widget class="QLabel" name="decodedInt">
   <property name="geometry">
    <rect>
     <x>741</x>
     <y>70</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="encodevarint">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>230</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
